site/examples/embeds.js

```diff
@@ -41,30 +41,12 @@ const Element = props => {
 
 const VideoElement = ({ attributes, children, element }) => {
   const editor = useEditor()
-  const selected = useSelected()
-  const focused = useFocused()
   const { url } = element
   return (
     <div {...attributes}>
       <div
         contentEditable={false}
```

Comment:
removed conditional rendering that was based on selection state. The input being selected is not treated as the element being selected. This might be a bug but I think it's tangential to this example.